persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAO.kt

```diff
@@ -5,16 +5,43 @@ import kotlinx.coroutines.flow.Flow
 
 data class MessageEntity(
     val id: String,
-    val content: String?,
+    val contentType: ContentType,
```

Comment:
Do we need to expose the `contentType`? From what I can see we never read it and always do a `when (content) {}`